tests/backup/backup_helper.go

```diff
@@ -1810,14 +1810,36 @@ func CreateBackupWithNamespaceLabel(backupName string, clusterName string, bkpLo
 	if err != nil {
 		return err
 	}
+	log.Infof("Getting the Uid of backup %v", backupName)
+	backupUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	if err != nil {
+		return err
+	}
+	backupInspectRequest := &api.BackupInspectRequest{
+		Name:  backupName,
+		Uid:   backupUid,
+		OrgId: orgID,
+	}
+
+	resp, err := Inst().Backup.InspectBackup(ctx, backupInspectRequest)
+	namespaceList := resp.GetBackup().GetNamespaces()
+	log.Infof("The list of namespaces backed up are %v", namespaceList)
+	if !reflect.DeepEqual(namespaceList, listOfLabelledNamespaces) {
+		return fmt.Errorf("list of namespaces backed up are %v which is not same as expected %v", namespaceList, listOfLabelledNamespaces)
+	}
+	backupLabels := resp.GetBackup().GetNsLabelSelectors()
+	log.Infof("The list of labels applied to backup are %v", backupLabels)
+	if !reflect.DeepEqual(backupLabels, namespaceLabel) {
+		return fmt.Errorf("labels applied to backup are %v which is not same as expected %v", backupLabels, namespaceLabel)
+	}
 	log.Infof("Successfully created backup [%s] with namespace label [%s]", backupName, namespaceLabel)
```

Comment:
Create a separate helper method for namespace and label verification for all backups